packages/react/src/composite/SegmentedControl/CHANGELOG.md

```
@@ -0,0 +1,7 @@
+# Changelog
+
+The SegmentedControl component is a composite component that allows us to select one or more options from a list. 
+
+## 2025-05-06 `1.0.0`
+
+- Initial release of the SegmentedControl component. ([#282](https://github.com/TiendaNube/nimbus-design-system/pull/282) by [@joacotornello](https://github.com/joacotornello))
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)